articles/libraries/lock/v11/migration-guide.md

```diff
@@ -0,0 +1,34 @@
+---
+section: libraries
+title: Migrating to Lock v11
+description: How to migrate to Lock v11
+---
+# Migrating to Lock v11
+
+Lock 11 is designed for **embedded login** scenarios. It operates with enhanced security and removes dependencies that have been deprecated as per Auth0's roadmap. In some cases, these security enhancements may impact application behavior when upgrading from an earlier version of Lock. 
+
+We recommend that instead of using Lock embedded in your application, you use **Centralized Login**, as it is the [most secure, powerful and flexible approach for authentication](/guides/login/centralized-vs-embedded).
+
```

Comment:
We should add a warning similar to what's found on /docs/libraries/auth0js/v9/migration-guide, mentioning that if you are using Lock to customize the Hosted Login Page, then you need to stick with v10. We have two scenarios now:

* If using Lock embedded in your application -> you must upgrade
* If using Lock in the customized Hosted Login Page (i.e. for centralized login) -> keep using Lock v10